core/platform/datastore/gae_datastore_services.py

```diff
@@ -278,15 +309,19 @@ class MockDatetimeType(type):
         """Pretends to be a datetime.datetime object."""
 
         def __instancecheck__(cls, other):
+            # type: (Any) -> bool
             """Validates whether the given instance is a datetime instance."""
             return isinstance(other, old_datetime_type)
 
+    # We use ‘ignore[misc]’ because MockDatetime subclasses a class created
+    # dynamically. These dynamic classes are not supported by mypy.
```

Comment:
I had a look at it closely. It might be fixed if we are able to get the types of `with_metaclass` as we will be removing python_utils and exposing future.utils directly. Should I mention it?

Reply:
Yup, give as much detail as possible in comments, especially when explaining "why".

Reply:
Done